src/index.css

```
@@ -185,165 +206,235 @@
+      0 6px 20px rgb(var(--accent-500) / 0.4);
+  }
+
+  .btn-primary:focus {
+    @apply ring-accent-500/50;
   }
-  .btn-primary:focus { @apply ring-accent-500/50; }
 
   /* Secondary — neutral glass */
   .btn-secondary {
     @apply bg-white/5 backdrop-blur-xl text-input-text;
     box-shadow: var(--glass-rim-subtle);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)